.github/workflows/docker-entropy-tester.yml

```diff
@@ -0,0 +1,50 @@
+name: Build and Push Entropy Tester Image
+on:
+  push:
+    tags:
+      - entropy-tester-v*
+  pull_request:
+    paths:
+      - "apps/entropy-tester/**"
+      - ".github/workflows/docker-entropy-tester.yml"
+      - "Dockerfile.node"
+  workflow_dispatch:
+    inputs:
+      dispatch_description:
+        description: "Dispatch description"
+        required: true
+        type: string
```

Comment:
Why is this argument here?

Reply:
I don't think you need the inputs, I think you just need `workflow_dispatch` with no children.  The `inputs` creates a UI where you can fill in values when doing the manual dispatch, but you aren't actually using the input anywhere so I think it should be removed